test/suite/controller/lintController.test.ts

```diff
@@ -111,6 +111,13 @@ describe("lintController.ts", async () => {
       const data = {
         success: true,
         messages: [
+          {
+            type: "error",
+            code: "no file",
+            message: "i have no file!",
```

Comment:
```suggestion
            message: "i am a general message",
```

IIRC we have 3 types of messages:

- general (apply to the entire add-on)
- global (apply to the entire file)
- specific (apply to a given line in a file)